rspace_client/client.py

```diff
@@ -74,7 +75,7 @@ def _get_formated_error_message(json_error):
     def _handle_response(response):
         # Check whether response includes UNAUTHORIZED response code
         if response.status_code == 401:
-            raise Client.AuthenticationError(response.json()['message'])
+            raise self.AuthenticationError(response.json()['message'])
```

Comment:
Ah, this isn't working because it's inside a `staticmethod`.  It still seems ugly referring to the class name inside the class though. I'll check this over another time